src/utils/tock.js

```diff
@@ -67,12 +67,12 @@ const get18FTockTruants = async (now, weeksAgo = 1) => {
 
   const tockUsers = await getCurrent18FTockUsers();
 
-  const allTruants = await getFromTock(
+  const allUnreportedUsers = await getFromTock(
```

Comment:
suggestion (non-blocking): I'm interpreting `allUnreportedUsers` as "users who have not been reported" as opposed to "users have not yet reported". Future me would find `allUntockedUsers` or `allLateTimeReporters` to be clearer.